torchmetrics/retrieval/recall_precision.py

```diff
@@ -0,0 +1,297 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
please rename this file to `precision_recall_curve.py`

Reply:
renamed 👌